src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,21 @@
 import React from 'react';
 import './MoviesList.scss';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[]
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <ul className="movies">
+    <li>
+      {movies.map((oneMovie: Movie) => (
+        <MovieCard
+          key={oneMovie.imdbId}
+          movie={oneMovie}
+        />
+      ))}
+    </li>
```

Comment:
```suggestion
    {movies.map((oneMovie: Movie) => (
      <li key={oneMovie.imdbId}>
        <MovieCard movie={oneMovie} />
      </li>
    ))}
```

Reply:
Not sure why you did it like that, isn't it more logical to put every movie card inside its 'li' instead of putting everything inside the only 'li'?

Reply:
@silchyn yeah, my bad, i dont know why))